src/main/app/form/Form.ts

```diff
@@ -119,7 +119,7 @@ export type FormFields = Record<string, FormField>;
 export type FormFieldsFn = (userCase: Partial<Case>, language?: string) => FormFields;
 
 export interface FormContent {
-  submit: {
+  submit?: {
```

Comment:
why is ? added to the interface?

Reply:
This is to allow the additional information to be included in one form whilst allowing the other form to be submitted without a submit button (as this is a hidden submitted value - I did not remove it altogether as this would be out of scope and may be relevant in future)

Reply:
(This prevents the additional information from being removed when the 'Upload' button is clicked and ensures the value is saved and passed to the CYA page)